app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,36 @@
   </head>
 
   <body>
+
+    <nav>
+
+      <% if current_member == nil %>
+        <div>
+          <a href="/sign_up">
+            Sign up
+          </a>
+        </div>
+    
+        <div>
+          <a href="/sign_in">
+            Sign in
+          </a>
+        </div>
+      <% else %>
+        <div>
+          <%= current_member.email %>
+        </div>
+        
+        <div>
+          <a href="/sign_out">
+            Sign out
+          </a>
+        </div>
+      <% end %>
+    </nav>
+
+    <p><%= notice %></p>
```

Comment:
I think the responsibility of modifying this file should be left up to the `draft:layout` generator. What do you think?

Reply:
I think that does make the most sense. It just feels annoying to have to type in the `sign_up`/`sign_in` routes after we run the generator, especially once we change them to `/member_sign_up`.

Reply:
@jelaniwoods Yes, that is annoying. Perhaps we should make a card to enhance `draft:layout` to be smarter; it can detect routes that include `/sign_up`, etc, and create links in the navbar for any that are present (even if there are multiple accounts, and/or it could be able to handle both `draft:account` and `devise` paths, etc).